### DISCUSSION TITLE
GitHub Future Ready Talent Repository

### DISCUSSION THREAD

**Discussion by TheSuranaverse** [github/india#353] [Students]
# Hey everyone! 👋

I have created a `Repository` which will contain all the `self-learning modules & notes` on them and the best part about it is that `@everyone` can contribute here. So like if you have completed a module & made some notes on it, you can `share` it here for everyone to learn & grow together. Since many of us don't have either a good connection or unlimited data/wifi, I hope this will be beneficial for them a lot.

- Currently, I have just uploaded the video of the `Orientation Session` which is of size less than `100 MB` for those who use **Mobile Data** to download & watch. Also if you don't want to watch then you can simply go through the `README.md` file to know about everything presented in the `Orientation Session` (thus **saving both time & data**). 

- Also for those who want to learn & complete this `INTERNSHIP` together, can join us here: https://discord.gg/ceMXzhfaka
It's a one-stop place for daily `Official FRT Announcement, Discuss & Solve Doubts, Events/Workshops + Hackathons + Internships/Jobs/Free Scholarship & Courses + 24/7 Live Music and much more...`

- Link of GitHub Repo: https://github.com/Neklaustares-tPtwP/Microsoft-Future_Ready_Talent-Internship

- Looking forward to more such contributions from all the amazing interns out there!

- Please do **UPVOTE** this discussion so that it can reach the maximum number of students out there.

# **THANK YOU!**

**Comment by Naga-Manohar-Y** [comment]
I registered in discord platform but i'm not able to access the discussion. what should I do now?

**Reply by TheSuranaverse** [reply]
Which discussion? 
Pls tell me your username there

**Reply by Naga-Manohar-Y** [reply]
Manohar

**Reply by TheSuranaverse** [reply]
@Naga-Manohar-Y I'm not able to find you on the discord server. Have you joined yet?

**Comment by BhavishaJhamnani28** [comment]
I have completed part 1 of the azure fundamentals but it still shows IN PROGESS

**Reply by TheSuranaverse** [reply]
@BhavishaJhamnani28 pls refer to the `Hot Queries and Solution` folder in the below link. There every query has been answered!
Link: https://github.com/Neklaustares-tPtwP/Microsoft-Future_Ready_Talent-Internship

To connect with us personally join us here: https://discord.gg/ceMXzhfaka

**Reply by Reshma-kumari-GitHub** [reply]
The organizers have mentioned in orientation that your progress will be updated in a weekly basis. hence wait for a week.

**Reply by TheSuranaverse** [reply]
@Reshma-kumari-GitHub that's true but we don't have to wait for a week, it will get updated by the 12th. For more official **FRT announcements** and services join the community here: https://discord.gg/ceMXzhfaka

**Comment by TheSuranaverse** [comment]
A group for all to get notified for any `Official FRT Announcement`, `Discuss & Clear any Doubts`, `Learn & Complete` together.
One-stop place for daily `Events/Workshops` + `Hackathons` + `Internships/Jobs` + `Free Scholarship, Courses, Swags & Goodies` + `24/7 Live Chat and Mentors to help you with your Doubts`  + `24/7 Live Music` & much more...

Currently, we are helping all the `Future Ready Talent` Interns clear their doubts & queries on a personal basis either on chat or voice, whichever they prefer. We will also be assisting them on Projects & help them complete this internship 🤗

Link to join: https://discord.gg/ceMXzhfaka
GitHub Solution Forum: https://github.com/github/india/discussions/353

Thank you 🙏

**Comment by ak258** [comment]
cant we just open each video of the course in a new tab and complete watching them faster if we already have prior knowlwdge of that course:?

**Reply by TheSuranaverse** [reply]
@ak258 no one knows for sure whether it will work or not or how they track our progress, but u can give it a try
I think it might work.

**Comment by Vijayalakshmi30** [comment]
@Neklaustares-tPtwP  Do you have any idea about how to create WordPress App? Please guide me to do it. I'm getting like the 
![Wordpress](https://user-images.githubusercontent.com/65505555/136753653-c59f7322-9bee-4dcb-8a9a-f34603d7fd33.png)
following image.

**Reply by TheSuranaverse** [reply]
Pls join the community & ping me there, we will explain you everything
Link to join: https://discord.gg/drQ7ySCdX2

**Comment by TheSuranaverse** [comment]
Hey everyone, go to the link below to get a solution for all your queries & doubts.

Don't forget to click on 🌟 on the top-most right corner of the page link to bookmark it for future reference.

Link: https://github.com/Neklaustares-tPtwP/Microsoft-Future_Ready_Talent-Internship/tree/master/Hot%20Queries%20%26%20Solution

**Comment by TheSuranaverse** [comment]
So helpful!

**Reply by TheSuranaverse** [reply]
yup

**Reply by TheSuranaverse** [reply]
Hi
